### ISSUE TITLE
Delivery object has wrong attributes

### ISSUE THREAD

**Issue by Anto59290** [recurly/recurly-client-python#213]
The Delivery class (used for gift cards) has the following attributes
```
attributes = (
        'address',
        'deliver_at',
        'email_address',
        'gifter_name',
        'method',
        'personal_message',
    )
```
If we refer to [documentation](https://dev.recurly.com/docs/gift-card-object) we should be able to fill in the first_name and last_name attributes:

> first_name:	string
> The first name of the recipient.
> last_name:	string
> The last name of the recipient.

**Comment by bhelx** [comment]
Thanks for reporting @Anto59290 
I'll try to get this fix out today.

**Comment by bhelx** [comment]
Closed with https://github.com/recurly/recurly-client-python/pull/214

This has been released in 2.6.1: https://github.com/recurly/recurly-client-python/releases/tag/2.6.1
